packages/react-combobox/src/stories/ComboboxDefault.stories.tsx

```diff
@@ -10,7 +10,9 @@ export const Default = (props: Partial<ComboboxProps>) => {
       <label id={comboId}>Best pet</label>
       <Combobox aria-labelledby={comboId} placeholder="Select an animal" {...props}>
         {options.map(option => (
-          <Option key={option}>{option}</Option>
+          <Option key={option} disabled={option === 'Ferret'}>
```

Comment:
Why should the Ferret be disabled? The hamster in the ball would be my choice. ;-)